docs/en/compute/HBase.md

```diff
@@ -69,12 +69,15 @@ Alternative ways are described in the [Advanced Setup]({{ '/en/compute/HBase.htm
 
 ## Example
 
-Start HBase:
+Ensure alluxio scheme is recognized before starting HBase:
 
 ```bash
 ${HBASE_HOME}/bin/start-hbase.sh
 ```
 
+If not, follow the [Usage FAQs](https://docs.alluxio.io/os/user/{{site.ALLUXIO_MAJOR_VERSION}}/en/operation/Troubleshooting.html#usage-faq)
```

Comment:
@wyykwan my bad. instead of the complete url we should use a relativized link similar to `{{ '/en/basic/Configuration-Settings.html' | relativize_url }}#configuration-sources`

Reply:
ohh, got it, thanks for clarifying!